### DISCUSSION TITLE
Definition of r_v??

### DISCUSSION THREAD

**Discussion by aninramesh** [ReedEspinosa/GSFC-Retrieval-Simulators#4] [Q&A]
Hi @ReedEspinosa,

I have a question regarding the r_v definition in canonicalCaseMap.py. In the definition of 'marine' aerosol, r_v is defined as follows: 
```
elif 'marine' in caseStr.lower():
  σ = [0.45, 0.70] # mode 1, 2,...
  rv = [0.2, 0.6]*np.exp(3*np.power(σ,2)) # mode 1, 2,... (rv = rn*e^3σ)
```
According to my understanding, rv = rn +3sigma^2

Am I missing anything?
![image](https://user-images.githubusercontent.com/20300514/181065330-96e4e47e-6c9e-43e2-83e1-e08419030b1e.png)

**Comment by ReedEspinosa** [comment]
I think the definition of `μ_v` in the table you attached is such that `rv=exp(μ_v)` (see definition on this [Wikipedia page](https://en.wikipedia.org/wiki/Log-normal_distribution)). Also, the formula in my comment should have `σ` squared `(rv = rn*e^3σ^2)`, but I  think that's separate from what you are asking about. Does that clarify, or am I missing something here?
